packages/node/src/generators/application/application.ts

```diff
@@ -198,16 +204,30 @@ function addProject(tree: Tree, options: NormalizedSchema) {
   }
   project.targets.serve = getServeConfig(options);
 
-  addProjectConfiguration(
-    tree,
-    options.name,
-    project,
-    options.standaloneConfig
-  );
+  if (options.isUsingTsSolutionConfig) {
+    writeJson(tree, joinPathFragments(options.appProjectRoot, 'package.json'), {
+      name: getImportPath(tree, options.name),
+      version: '0.0.1',
+      private: true,
+      nx: {
+        name: options.name,
+        projectType: 'application',
+        sourceRoot: project.sourceRoot,
+        targets: project.targets,
+        tags: project.tags,
```

Comment:
Tags could still be empty
```suggestion
        tags: project.tags?.length ? project.tags : undefined,
```